### PR TITLE
Add lambda function for generating profile plots

### PR DESCRIPTION
**Pull request by Bobfrat**
@mwengren FYI

### CODE REVIEW

**Comment by Bobfrat** [comment]
Addresses #36

File: aws/lambda/profile_plots.py [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+    thredds_url = event['dap']
+    deployment_name = thredds_url.split('/')[7]
+
+    with netCDF4.Dataset(thredds_url, 'r') as nc:
+        for parameter in PARAMETERS:
+            filename = '{}/{}.png'.format(deployment_name, parameter)
+            try:
+                plot_from_nc(deployment_name, nc, parameter, filename)
+            except Exception:
+                print("Failed to generate plot for %s" % parameter)
```
Comment (benjwadams) [inline aws/lambda/profile_plots.py]: Use `logger.exception` here instead and send to stdout.
Don't use `print` for anything that should be logged.

**Comment by benjwadams** [comment]
We should add tests to these functions.  I'll merge once the print statements are converted over to log statements, but plots have broken in the past due to library changes.  It'd be good to cover the AWS portions with mocks, etc and simulate failure conditions.

**Comment by Bobfrat** [comment]
This code is not actually executed, just wanted to get our work from the code sprint into this open source repo as per micahs request. In general, we need to think about a logging strategy for lambda functions.

**Pull request merged**